content/content-management/multilingual.md

```diff
@@ -43,7 +43,7 @@ title = "Mon blog"
 weight = 2
 [Languages.fr.params]
 linkedin = "lien-francais"
-[Languages.fr.navigation]
```

Comment:
I think the correct fix here is to remove the params above, i.e.  [Languages.fr.params] >  [Languages.fr]